src/main/java/it/gov/pagopa/pu/classification/repository/view/PaymentsReportingViewRepository.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.pu.classification.repository.view;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import it.gov.pagopa.pu.classification.model.view.PaymentsReportingView;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.Repository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+import java.time.LocalDate;
+
+@RepositoryRestResource(path = "payments-reporting-view")
+public interface PaymentsReportingViewRepository extends Repository<PaymentsReportingView, String> {
+
+  @Query("SELECT distinct p FROM PaymentsReportingView p "
+    + "WHERE p.organizationId = :organizationId "
+    + "AND (:iuf IS NULL OR p.iuf = :iuf) "
+    + "AND (:regulationUniqueIdentifier IS NULL OR p.regulationUniqueIdentifier = :regulationUniqueIdentifier) "
+    + "AND ((cast(:fromDate AS DATE) IS NULL AND cast(:toDate AS DATE) IS NULL) "
+    + "OR (cast(:fromDate AS DATE) IS NOT NULL AND cast(:toDate AS DATE) IS NOT NULL "
+    + "AND p.regulationDate BETWEEN :fromDate AND :toDate))")
```

Comment:
regulationDate is wrong
```suggestion
    + "AND (:regulationDateFrom IS NULL OR p.regulationDate >= cast(:regulationDateFrom AS DATE)) "
    + "AND (:regulationDateTo IS NULL OR p.regulationDate <= cast(:regulationDateTo AS DATE)) "
```